go.mod

```diff
@@ -7,20 +7,21 @@ module github.com/dell/csi-vxflexos/v2
 
 //replace github.com/dell/goscaleio => ./goscaleio
 
-replace github.com/dell/gocsi => ./gocsi
+// replace github.com/dell/gocsi => ./gocsi
```

Comment:
Can we put this line back in?
We modified gocsi; so building/testing should be done with the latest

Reply:
done